common/src/androidTest/java/com/microsoft/identity/common/MsalOAuth2TokenCacheTest.java

```diff
@@ -344,6 +389,28 @@ public void saveTokensWithAggregationSingleEntry() throws ClientException {
         assertNotNull(entry.getRefreshToken());
     }
 
+    @Test
+    public void saveTokensWithAggregationSingleEntryWithMalformedDataInCache() throws ClientException {
+        // Prepopulate the cache with unparseable, junk data
+        mSharedPreferencesFileManager.putString(JUNK_KEY, JUNK_VALUE);
+
+        final List<ICacheRecord> result = loadTestBundleIntoCacheWithAggregation(
+                defaultTestBundleV2
+        );
+
+        assertEquals(1, result.size());
+
+        final ICacheRecord entry = result.get(0);
+
+        assertNotNull(entry.getAccount());
+        assertNotNull(entry.getIdToken());
+        assertNotNull(entry.getAccessToken());
+        assertNotNull(entry.getRefreshToken());
```

Comment:
Should we check that these have some particular value?

Reply:
Refined in 895c3d2c36748b8a7c40d6664bd472f11d22bf52